go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/googleinterns/terraform-cost-estimation
 
 go 1.15
 
-require github.com/hashicorp/terraform-json v0.5.0 // indirect
+require github.com/hashicorp/terraform-json v0.5.0 // indirect
```

Comment:
Nit: the change to this file looks unrelated.

Reply:
Yeah, not sure why it appears as a change.